docs/application/dotnet/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1099 @@
+    }
+    ```
+
+    To retrieve the commands, you have to add the command list:
+
+    -   You can use the `GetCurrentCommands()` method of the [Tizen.Uix.VoiceControlManager.VoiceControlManagerClient](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.VoiceControlManagerClient.html) class to get all commands from the command list:
+
+        ```csharp
+        void ForeachCommand()
+        {
+            try
+            {
+                IEnumerable<VoiceCommand> command = VoiceControlManagerClient.GetCurrentCommands();
+            }
+            catch (Exception e)
+            {
+                /// Error handling
+            }
+        }
+        ```
+
+    -   You can use the `Commands` property of the [Tizen.Uix.VoiceControlManager.VoiceCommandsGroup](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.VoiceCommandsGroup.html) class to get the current commands in an output parameter.
+        ```csharp
+        void GetCommands()
+        {
+            try
+            {
+                VoiceCommand current_command = new VoiceCommand();
+                VoiceCommandsGroup group = new VoiceCommandsGroup();
+
+                /// Other actions
+
+                current_commands = group.Commands;
+            }
+            catch (Exception e)
+            {
+                /// Error handling
+            }
+        }
+        ```
+
+4.  Register the commands for recognition by setting the command group to the voice control service.
+
+    If you want to update the registered commands, set the command group again with the updated commands:
+
+    ```csharp
+    void SetCommand()
+    {
+        try
+        {
+            VoiceCommandsGroup group = new VoiceCommandsGroup();
+            VoiceCommand command = new VoiceCommand();
+            command.Command = "test";
+            command.Type = CommandType.Foreground;
+
+            group.Commands.Add(command);
+
+            VoiceControlManagerClient.SetCommands(group);
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+5.  When no longer needed, unset the command group:
+
+    ```csharp
+    /// Unset the command group
+    void UnsetCommand()
+    {
+        try
+        {
+            VoiceCommandsGroup group = new VoiceCommandsGroup();
+            VoiceCommand command = new VoiceCommand();
+            command.Command = "test";
+            command.Type = CommandType.Foreground;
+
+            group.Commands.Remove(command);
+
+            VoiceControlManagerClient.SetCommands(group);
+            ///or
+            VoiceControlManagerClient.ClearCommands();
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+
+<a name="register_command"></a>
+## Register Command
+
+- You can register the recognition commands by setting a file to voice control service, and the file is including the required commands.
```

Comment:
@mijin85cho, I agree that the original sentence is concise. However, the confusion was due to the term "file".

Please let me know your view on the following suggestion:

Reply:
@surbhig12 I think you didn't get my point...
I asked you to check the sentence because the sentence is too lengthy. As I asked you, I hope to make the sentence concise, not making it lengthier with adding another example. The examples we suggest in the comment history is to make you understand, and I don't think adding them to the actual review result is recommended.. If you want to add more description related with the examples we provided during the review, how about rather using words used in the sample code like below?

You can register the commands for recognition by setting the command group to the voice control service from a file that includes the commands. The parameter `commandFilePath` is used to get the path of the file.

Reply:
@mijin85cho, I understood that the sentence should be concise and thats why I agree with the original sentence:

Register the commands for recognition by setting the command group to the voice control service from file.

the doubt was arising because of the term "file". To address this. your suggestion is clear:

You can register the commands for recognition by setting the command group to the voice control service from a file that includes the commands. The parameter `commandFilePath` is used to get the path of the file.

Reply:
Thanks for confirming @surbhig12

Reply:
@surbhig12 @mijin85cho Done Thank you.